sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -673,6 +673,19 @@ case class Window(
   def windowOutputSet: AttributeSet = AttributeSet(windowExpressions.map(_.toAttribute))
 }
 
+case class SessionWindow(
```

Comment:
Ah OK this is conflicting with SessionWindowExpression when renaming. OK to leave as it is.